app/helpers/address_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module AddressHelper
+  def postcode_lookup(postcode)
+    postcode_url = "#{ENV['API_URL']}?postcode=#{postcode}&key=#{ENV['API_KEY']}"
```

Comment:
Would it be possible to make the environment variable names a little more descriptive, so we know what they are for when looking in the PaaS console?  E.g. `OS_PLACES_API_URL` and `OS_PLACES_API_KEY`.

Reply:
Thanks @brucebolt - I've updated this to `ORDNANCE_SURVEY_PLACES_API_KEY`